### PR TITLE
Cognitive Services PowerShell Update to RP Version 2022-03-01

### PR DESCRIPTION
**Pull request by yangyuan**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1159
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+# Get-AzCognitiveServicesAccountModel
+
+## SYNOPSIS
+Get Models available for a Cognitive Services account
+
+## SYNTAX
+
+### DefaultParameterSet (Default)
+```
+Get-AzCognitiveServicesAccountModel [[-ResourceGroupName] <String>] [-AccountName] <String>
```
Comment (BethanyZhou) [inline src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md:17]: How about name Get-AzCognitiveServicesAccountModel to Get-AzCognitiveServicesAccount? And change AccountName to Name and keep AccountName as alias

File: src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## DESCRIPTION
+Get Models available for a Cognitive Services account
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzCognitiveServicesAccountModel -ResourceGroupName cognitive-services-resource-group -AccountName resource-name
+```
```
Comment (BethanyZhou) [inline src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md:35]: Please provide sample output here

File: src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+### System.String
+
+## OUTPUTS
+
+### Microsoft.Azure.Management.CognitiveServices.Models.AccountModel
+
+## NOTES
+
+## RELATED LINKS
```
Comment (BethanyZhou) [inline src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md:114]: Please add related links.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md (hunk 1 above)
Comment (yangyuan) [reply src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md:17]: We have "Get-AzCognitiveServicesAccount"
"Get-AzCognitiveServicesAccountModel" is right now used for listing models available for this account.
Though it is uncertain right now, we do have potential to support individual Get in future, we want to reserve the "Name" parameter for this. Just like what we do in "Get-AzCognitiveServicesAccountDeployment" and "Get-AzCognitiveServicesAccountCommitmentPlan".

File: src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md (hunk 2 above)
Comment (yangyuan) [reply src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md:53]: Added

File: src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md (hunk 3 above)
Comment (yangyuan) [reply src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md:132]: Added, note there is no commands have direct relation with this command.
But when creating Deployment, customer might want to know the details of model. Thus I added `New-AzCognitiveServicesAccountDeployment` as related

File: src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md (hunk 1 above)
Comment (BethanyZhou) [reply src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md:17]: Make sense.

**Review by BethanyZhou — APPROVED** [review]
Looks good to me. Hold on this PR as current is code freeze time

**Pull request merged**